notification/shoutrrr.go

```diff
@@ -2,14 +2,45 @@ package notification
 
 import (
 	"fmt"
+	"net/url"
+	"os"
 	"strings"
 
 	"github.com/containrrr/shoutrrr"
 	"github.com/containrrr/shoutrrr/pkg/types"
 	"github.com/flanksource/commons/collections"
+	"github.com/flanksource/commons/utils"
 	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/mail"
 )
 
+// defaultSMTPPrefix indicates that the shoutrrr URL for smtp should use
+// the system's SMTP credentials.
+const defaultSMTPPrefix = "smtp://system/"
```

Comment:
Can we create a SendEmail(ctx, title, message, properties) that uses this default? and then replace all other usages of SMTP_* with this method ?